polkadot-host-spec/c06-consensus.tm

```diff
@@ -997,6 +1005,24 @@
 
   <subsection|GRANDPA Messages Specification>
 
+  <\definition>
+    <label|defn-gossip-message><strong|Gossip Message> is a variant as
```

Comment:
Is this Grandpa specific, i.e. we only have a Gossip Message for Granpa then better calling it GRANDPA gossip. If it is used somewhere else then better define it in the network section.

Reply:
Done!